### PR TITLE
chore: Improve unit test coverage on adding test coverage on CryptoCreateHandler 

### LINKED ISSUE #13234: CryptoCreateHandler [76.43 %]
Increase Unit Test Code coverage for CryptoCreateHandler

### PR DESCRIPTION
**Pull request by thomas-swirlds-labs**
**Description**:
Increase test coverage > 80% on CryptoCreateHandler

**Related issue(s)**:

Fixes #13234

### CODE REVIEW

File: hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/CryptoCreateHandlerTest.java [hunk 1]
```diff
@@ -256,6 +279,39 @@ void preHandleWorksWhenInitialBalanceIsZero() throws PreCheckException {
...
+        final byte[] evmAddress = CommonUtils.unhex("6aeb3773ea468a814d954e6dec795bfee7d76e26");
+        txn = new CryptoCreateBuilder()
+                .withAlias(Bytes.wrap(evmAddress))
+                .withStakedAccountId(3)
+                .build();
+        final var context = new FakePreHandleContext(readableStore, txn);
+        subject.pureChecks(txn);
+        subject.preHandle(context);
+
+        assertEquals(txn, context.body());
```
Comment (mhess-swl) [inline hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/CryptoCreateHandlerTest.java:294]: I'm seeing this in a few places. Preference is to favor assertj, especially for new lines of code

File: hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/CryptoCreateHandlerTest.java [hunk 2]
```diff
@@ -256,6 +279,39 @@ void preHandleWorksWhenInitialBalanceIsZero() throws PreCheckException {
...
+    @DisplayName("preHandle fails when invalid alias key")
+    void preHandleWorksWhenHasAlias() throws PreCheckException {
+        final Bytes SENDER_ALIAS =
+                Bytes.fromHex("3a21030edcc130e13fb5102e7c883535af8c2b0a5a617231f77fd127ce5f3b9a620591");
+        txn = new CryptoCreateBuilder()
+                .withAlias(SENDER_ALIAS)
+                .withStakedAccountId(3)
+                .build();
+        final var context = new FakePreHandleContext(readableStore, txn);
+        subject.pureChecks(txn);
```
Comment (mhess-swl) [inline hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/CryptoCreateHandlerTest.java:309]: What's the purpose of calling `pureChecks(...)` here?

Comment (thomas-swirlds-labs) [reply hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/CryptoCreateHandlerTest.java]: Good catch 👍  
Removed

**Review by mhess-swl — APPROVED** [review]
LGTM!

**Pull request merged**